DAL/EmployeeDao.cs

```diff
@@ -7,8 +8,48 @@ public class EmployeeDao : BaseDao
     {
         public async Task<List<Employee>> GetAllEmployeesAsync()
```

Comment:
Please, rename the method and add a comment on what info is fetched here. See Naz's example.